From 2b55524fab066cb14880a819f570b7f839bae1ce Mon Sep 17 00:00:00 2001
From: Jim Somerville <Jim.Somerville@windriver.com>
Date: Wed, 24 Apr 2019 22:44:12 -0400
Subject: [PATCH] glusterfs-api: remove unnecessary library options

These cause QA errors when users of the api such
as qemu generate their packages.  The reported
error is unnecessary or redundant RPATH specified.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Jim Somerville <Jim.Somerville@windriver.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
[modify patch for release-6.1]
Signed-off-by: Hongzhi.Song <hongzhi.song@windriver.com>
---
 glusterfs-api.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glusterfs-api.pc.in b/glusterfs-api.pc.in
index 4a2edb7..153c4ad 100644
--- a/glusterfs-api.pc.in
+++ b/glusterfs-api.pc.in
@@ -8,5 +8,5 @@ Description: GlusterFS API
 /* This is the API version, NOT package version */
 Version: @GFAPI_VERSION@
 Requires: @PKGCONFIG_UUID@
-Libs: -L${libdir} @GFAPI_LIBS@ -lgfapi -lglusterfs -lgfrpc -lgfxdr
+Libs: -lgfapi -lglusterfs -lgfrpc -lgfxdr
 Cflags: -I${includedir} -D_FILE_OFFSET_BITS=64 -D__USE_FILE_OFFSET64 -D__USE_LARGEFILE64 -DUSE_POSIX_ACLS=@USE_POSIX_ACLS@
-- 
2.8.1

